v4/themes.css

```diff
@@ -65,110 +65,37 @@ a#searchLink {
 
 
 /*** Home-specific (i.e. not used across pages) ***/
-.default-card-header {
+.iacrMain-cardHeader {
   background: #04269E;
   color: #f0f0f0 !important;
 }
-.default-card-header a {
-  color: #f0f0f0 !important;
-}
 
-.default-card-footer {
-  background: #E9EBF3 !important;
+.iacrMain-cardHeader a {
```

Comment:
you changed footer to header here?

Reply:
No, just deleted. The way github displays it is weird.